server/test/e2e/asset.e2e-spec.ts

```diff
@@ -788,4 +791,55 @@ describe(`${AssetController.name} (e2e)`, () => {
       expect(asset.stack).toEqual(expect.arrayContaining([expect.objectContaining({ id: asset3.id })]));
     });
   });
+
+  describe(AssetRepository.name, () => {
```

Comment:
This doesn't seem like the right place for these tests, but I also can't be bothered :smile:. These tests are better than no tests I guess. I think we may delete them in the future if we ever write proper e2e tests for the microservices code.

Reply:
I agree entirely. No attachment to these tests - only the feeling that something is better than nothing.